generator/testdata/google/example/library/v1/library_service.gapic.cc.baseline

```diff
@@ -12,6 +12,12 @@ LibraryService::CreateBook(
 ::google::example::library::v1::CreateBookRequest const& request) {
   // TODO: actual useful work, e.g. retry, backoff, metadata, pagination, etc.
```

Comment:
nit: is this still applicable?  Looks like you are doing useful work...

Reply:
Outdated comment, will remove.